src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -134,17 +134,18 @@ def upload(
         >>> meshing_session.upload(file_name=mesh_file_name, remote_file_name="elbow.msh.h5")
         >>> meshing_session.meshing.File.ReadMesh(FileName="elbow.msh.h5")
         """
-        local_file_name = pathlib.Path(file_name)
-        if local_file_name.exists() and local_file_name.is_file():
-            if remote_file_name:
-                shutil.copyfile(
-                    file_name,
-                    str(self.fluent_cwd / f"{os.path.basename(remote_file_name)}"),
-                )
-            else:
-                shutil.copyfile(
-                    file_name, str(self.fluent_cwd / f"{os.path.basename(file_name)}")
-                )
+        files = _get_files(file_name)
+        for file in files:
+            if file.exists() and file.is_file():
```

Comment:
@hpohekar Redundant call to `file.exists()` can be removed.

Reply:
Removed. Thank you.